CREDITS

```diff
@@ -126,8 +126,11 @@ of the Coq Proof assistant during the indicated time:
   Sébastien Hinderer (INRIA, 2014)
   Gérard Huet (INRIA, 1985-1997)
   Matej Košík (INRIA, 2015-2017)
+  Leonidas Lampropoulos (University of Pennsylvania, 2018)
   Pierre Letouzey (LRI, 2000-2004, PPS, 2005-2008,
                    INRIA-PPS then IRIF, 2009-now)
+  Yao Li (ORCID: https://orcid.org/0000-0001-8720-883X,
+          University of Pennsylvania, 2018)
```

Comment:
@mattam82, or @maximedenes, or @zimmi48, could you remind the rules about changing the CREDITS file?

Reply:
The author of this PR is among the rare contributors showing that they have read the CONTRIBUTING guide by adding this line:

>If it's your first significant contribution to Coq (significant means: more than fixing a typo), your pull request should include a commit adding your name to the CREDITS file (possibly with the name of your institution / employer if relevant to your contribution, an ORCID if you have one —you may log into https://orcid.org/ using your institutional account to get one—, and the year of your contribution).

Reply:
:laughing: